worlds/musedash/docs/en_Muse Dash.md

```diff
@@ -17,7 +17,7 @@ The goal of Muse Dash is to collect a number of **Music Sheets**. Once you've co
 
 Only the base Muse Dash game is required in order to play this game.
 
-However, the **Just as Planned DLC** is recommended as the number of possible songs increases from 60+ to 400+ songs, which adds to the variety and increases replayability.
+However, the **[Just as Planned]**/**[Muse Plus]** DLC is recommended, as they increase the number of possible songs from 60+ to 400+ songs, which adds to the variety and increases replayability.
```

Comment:
```suggestion
However, the **[Just as Planned]**/**[Muse Plus]** DLC is recommended, as it increases the number of possible songs from ~60 to 400+ songs, which adds to the variety and increases replayability.
```
This fixes a singular/plural error and changes "60+" to "~60" since "60+" could be almost any number. Could also use `≈` depending on preference

Reply:
I'll stick with ~ as ≈ is kinda a match term some people might not know of.